### ISSUE TITLE
Acurite 00275RM (#74) stay NA on versions higher than 0.19b 

### ISSUE THREAD

**Issue by mihasik22** [thejeffreystone/hassio_addons#52]
After updating to any version higher than 0.19b both outdoor sensors acurite 00275RM (code 74) become unavaliable.
Data from sensors transmitted to mqtt but the home assistant considers that these devices do not exist.
I try almost everything but none happened - only works on version 0.19b and below.

Versions logs look different, but in detailed research i saw only one difference in payload: 0.19 reports rssi with 6 digits precision, and newest versions only 5. 

Old version log:
Received message: rtl_433/events : {"time": "2023-06-06 17:17:22+1000", "protocol": 74, "model": "Acurite-00275rm", "subtype": 2, "id": 8468, "battery_ok": 1, "temperature_C": 33.2, "humidity": 30, "temperature_1_C": 25.0, "mic": "CRC", "mod": "ASK", "freq": 433.97594, "rssi": -0.196003, "snr": 18.48466, "noise": -18.6807}

New version log:
DEBUG:Received Device Data from SDR and sent to MQTT: rtl_433/events : {"time": "2023-06-06 14:10:57+1000", "protocol": 74, "model": "Acurite-00275rm", "subtype": 2, "id": 8468, "battery_ok": 1, "temperature_C": 37.7, "humidity": 30, "temperature_1_C": 23.8, "mic": "CRC", "mod": "ASK", "freq": 433.92842, "rssi": -8.39855, "snr": 16.93324, "noise": -25.3318}

If you need any additional information to find the error, I am ready to provide it.

**Comment by thejeffreystone** [comment]
Yeah this one should be fixed. I hope. Its been. while since I dedicated any time to this add-on...
